schema/README.md

```diff
@@ -1,16 +1,11 @@
 # Schema Generator
 
 This folder contains a script for generating enigma-go based on a specific version of Qlik Associative Engine.
-The version to be used must be one of the published versions of the Qlik Associative Engine, see [here](https://hub.docker.com/r/qlikcore/engine/tags/).
+The Engine OpenRPC specification can be found in the schema folder `./schema/engine-rpc.json`.
 
-Please note that to be able to generate enigma-go you will need to accept the [EULA](https://core.qlik.com/eula/).
-
-```bash
-ACCEPT_EULA=<yes/no> ENGINE_VERSION=<version> ./schema/generate.sh
-```
-
-If a version is not specified the script will default to the latest published version.
-
-```bash
-ACCEPT_EULA=<yes/no> ./schema/generate.sh
+If you want to manually generate a specific version of the Engine specification:
+- download the spec* and save it as `engine-rpc.json` in the `./schema` folder
```

Comment:
Is there a URL on qlik.dev where the spec can be downloaded from? Even if it is not a direct link it would still be valuable info to add.

Reply:
Turns out it's exposed through the ingress so you could go to `https://<tenant>.<region>.qlikcloud.com/api/engine/openrpc` :D

Reply:
With an API-key you could fetch it from QCS at: `https://<tenant>.<region>.qlikcloud.com/api/engine/openrpc`

Reply:
👍  - the download QIX spec link is finally available on qlik.dev